sdk/Lusid.Sdk.Tests/LusidApiFactoryTests.cs

```diff
@@ -12,6 +12,7 @@
 using Newtonsoft.Json;
 using NUnit.Framework;
 using System.Net;
+using RestSharp;
```

Comment:
Is this necessary?

Reply:
Needed for the ResponseStatus enum access